integration_tests/src/main/python/string_test.py

```diff
@@ -483,11 +483,14 @@ def test_regexp_replace():
                 'regexp_replace(a, "a|b|c", "A")'),
             conf={'spark.rapids.sql.expression.RegExpReplace': 'true'})
 
+@pytest.mark.xfail(condition=is_databricks_runtime(),
```

Comment:
Do we want to xfail this or simply skip it?  The point of xfailing is to track tests that we eventually expect to pass.  I think skip is more appropriate for this one.

Reply:
Skipping it makes sense to me. Should I also update `test_concat_ws_sql_col_sep_only_sep_specified` in the same file to skip instead of xfail? (that's where I copied the pattern from).

Reply:
Yes, any test that we are never expecting to pass because it's an invalid premise for testing in a particular environment should be skipped rather than xfailed.  We should reserve xfail for tests we think should pass at some point, as they essentially serve as TODO's in the test output.